support/cas-server-support-pac4j/src/main/java/org/apereo/cas/support/pac4j/config/support/authentication/Pac4jAuthenticationEventExecutionPlanConfiguration.java

```diff
@@ -125,6 +125,7 @@ private void configureFoursquareClient(final Collection<BaseClient> properties)
     private void configureGoogleClient(final Collection<BaseClient> properties) {
         final Pac4jProperties.Google google = casProperties.getAuthn().getPac4j().getGoogle();
         final Google2Client client = new Google2Client(google.getId(), google.getSecret());
+        client.setName("Google");
```

Comment:
@mmoayyed I created #2826 for the master branch.

Do you want the name externalised before integrating this change?  If so what's the best way to do so? via config as a name attribute?  via messages.properties?